<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorAtemCutWidget</class>
 <widget class="QWidget" name="InspectorAtemCutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxStep">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxMixerStep">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTriggerOnNext">
     <property name="text">
      <string>Trigger on next</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelStep">
     <property name="text">
      <string>Step</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelMixerStep">
     <property name="text">
      <string>M / E</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxMixerStep</tabstop>
  <tabstop>comboBoxStep</tabstop>
  <tabstop>checkBoxTriggerOnNext</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxStep</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorAtemCutWidget</receiver>
   <slot>stepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorAtemCutWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stepChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
 </slots>
</ui>
